src/components/Chip/Chip.scss

```diff
@@ -0,0 +1,6 @@
+.Chip {
+    font-size: var(--font-size-small);
+    padding: var(--spacing-2x) var(--spacing-5x);
+    background-color: var(--color-neutral);
```

Comment:
Let's make the background color a bit lighter, I checked and I think `--color-neutral-25` should be good